qa/L0_model_update/instance_update_test.py

```diff
@@ -510,11 +510,11 @@ def test_instance_resource_decrease(self):
             if os.environ["RATE_LIMIT_MODE"] == "execution_count":
                 # Make sure the previous max resource limit of 4 is reduced to 3
                 # when no explicit limit is set.
-                self.assertIn("Resource: R1\t Count: 3", f.read())
+                self.assertIn("Resource: R1\\t Count: 3", f.read())
             else:
                 # Make sure the max resource limit is never set to 3 when
                 # explicit limit of 10 is set.
-                self.assertNotIn("Resource: R1\t Count: 3", f.read())
+                self.assertNotIn("Resource: R1\\t Count: 3", f.read())
```

Comment:
Note for possible follow-up, if these end up poorly escaped, maybe we can consider embedding some something like 4 spaces "    " literally in the string rather than using `\t`.

Reply:
was also thinking these could be a table - or a json instead